server/i18n/be.json

```diff
@@ -2875,6 +2875,10 @@
     "id": "app.webhooks.get_incoming_by_user.app_error",
     "translation": "Не ўдалося атрымаць вэбхук."
   },
+  {
```

Comment:
the translations are meant to be only made in the en.json file.

Reply:
Good point. I reverted the changes to the other .json files.